doc/sphinx-guides/source/developers/sql-upgrade-scripts.rst

```diff
@@ -28,7 +28,9 @@ How to Create a SQL Upgrade Script
 
 We assume you have already read the :doc:`version-control` section and have been keeping your feature branch up to date with the "develop" branch.
 
-Create a new file called something like ``V4.11.0.1__5565-sanitize-directory-labels.sql`` in the ``src/main/resources/db/migration`` directory. Use a version like "4.11.0.1" in the example above where the previously released version was 4.11, ensuring that the version number is unique. Note that this is not the version that you expect the code changes to be included in (4.12 in this example). When the previously released version is a patch version (e.g. 5.10.1), use "5.10.1.1" for the first SQL script version (rather than "5.10.1.0.1"). For the "description" you should the name of your branch, which should include the GitHub issue you are working on, as in the example above. To read more about Flyway file naming conventions, see https://documentation.red-gate.com/fd/migrations-184127470.html
+Create a new SQL file in the ``src/main/resources/db/migration`` directory. Name the file to something similar to ``V6.1.0.1.sql``. In this example ``6.1`` represents the current version of Dataverse, with the last digit representing number of the script for that version. Should a newer version be merged while you work on your pull request (PR), you must update your script to a higher number.
+
+Previously, we used the Flyway file naming conventions. However, this approach occasionally led to inadvertent merging of multiple scripts with the same version, such as ``6.0.0.1__0000-wonderful-pr.sql`` and ``6.0.0.1__0001-wonderful-pr.sql``. After careful consideration and analysis, we agreed to adopt the convention mentioned earlier for naming files. This helps us detect conflicts before merging a PR, preventing the development branch from breaking an installation. Additionally, we request that each user writes a meaningful comment on the first line. For more information on Flyway file naming conventions, see https://documentation.red-gate.com/fd/migrations-184127470.html 
```

Comment:
```suggestion
Previously, we used longer, more descriptive file naming conventions supported by Flyway. However, this approach occasionally led to inadvertent merging of multiple scripts with the same version, such as ``V6.0.0.1__0000-wonderful-pr.sql`` and ``V6.0.0.1__0001-lovely-pr.sql`` where ``V6.0.0.1`` must be unique. After careful consideration, we agreed to adopt the convention mentioned above for naming files. This helps us detect conflicts before merging a PR, preventing the develop branch from being undeployable due to a Flyway conflict. For more information on Flyway file naming conventions, see https://documentation.red-gate.com/fd/migrations-184127470.html 
```

I'm trying to communicate that the first part is what matters, the `V6.0.0.1` part can conflict.

Also, I removed the part about the comment from the history lesson and put it higher up under the "do it this way" section. 😄